addons/website/views/snippets/s_cards_soft.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_cards_soft" name="Cards Soft">
+    <section class="s_cards_soft o_cc o_cc1 pt48 pb32">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12">
+                    <h2 style="text-align: center;">Your Journey Begins Here</h2>
+                    <p class="lead" style="text-align: center;">We make every moment count with solutions designed just for you.</p>
+                </div>
+                <div data-name="Card" class="col-lg-4 pt16 pb16">
+                    <div class="s_card card rounded o_cc o_cc2" data-vxml="001" data-name="Card" data-snippet="s_card" style="border-width: 0 !important;">
```

Comment:
I don't think that we need `data-name="Card"` here, as it will never be visible in the editor. The one that will be visible is the one from the parent col